src/main/java/io/github/moonslanding/tlm/engine/SpritedGameObject.java

```diff
@@ -25,7 +25,8 @@ public SpritedGameObject(int x, int y, String spriteName) {
     public void render(Graphics graphics) {
         if (sprite != null) {
             BufferedImage tintApplied = applyTint(sprite.getTexture());
-            graphics.drawImage(tintApplied, getX(), getY(), null);
+            BufferedImage scaled = applyZoomFactor(tintApplied, 3); // apply zoom factor here
```

Comment:
should `zoomFactor` not be constant?

Reply:
yep

Reply:
should we do zoomfactor private attribute and set them?